source/maintainer-guide/installing/map-print-setup.rst

```diff
@@ -13,15 +13,24 @@ if your network requires a webproxy to be set up to access the internet.
 
 Locate the file `WEB-INF/config-print/print-config.yaml`, this configuration file has a lot of options to customise the print options. Read more about the various parameters at http://www.mapfish.org/doc/print/configuration.html 
 
-
 The folder contains 3 template files:
 
 - `template.pdf` and `template-landscape.pdf` which are used to generate the map viewer pdf
 
 - `template-thumbnail.pdf` which is used to build a thumbnail in the metadata editor (see :ref:`linking-thumbnail-from-wms`).
 
-
 These templates are created by exporting pdf from the included `template.odf` file in the folder. 
 
+Creation of PDF uses some advanced java features which have some prerequisites.
```

Comment:
Maybe we could add the exception so search engine will index the error and point to the solution?